src/runtime/button-api.js

```diff
@@ -60,10 +75,14 @@ export class ButtonApi {
 
   /**
    * @param {!../model/doc.Doc} doc
+   * @param {!function(!../api/client-event-manager-api.ClientEvent)} logEventFunc
    */
-  constructor(doc) {
+  constructor(doc, logEventFunc) {
```

Comment:
NIT: I've seen "function" abbreviated as Fun many many times. You can leave it func if you want though.